components/dashboard/src/workspaces/VideoCarousel.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import React, { useState } from "react";
+import { trackVideoClick } from "../Analytics";
+
+import "lite-youtube-embed/src/lite-yt-embed.css";
+import "lite-youtube-embed/src/lite-yt-embed";
+
+interface Video {
+    id: string;
+    title: string;
+    analyticsLabel: string;
+}
+
+const videos: Video[] = [
+    { id: "1ZBN-b2cIB8", title: "Gitpod in 120 seconds", analyticsLabel: "gitpod-demo" },
+    { id: "zhZNnzFlZnY", title: "Getting started with Gitpod", analyticsLabel: "getting-started-with-gitpod" },
+    { id: "kuoHM2bpBqY", title: "Fully automate your dev setup", analyticsLabel: "automate-gitpod-setup" },
+    { id: "_CwFzCbAsoU", title: "Personalise your workspace", analyticsLabel: "personalise-gitpod-workspace" },
+];
+
+declare global {
+    namespace JSX {
+        interface IntrinsicElements {
+            "lite-youtube": any;
+        }
+    }
+}
+
+export const VideoCarousel: React.FC = () => {
+    const [currentVideo, setCurrentVideo] = useState(0);
+
+    const handleDotClick = (index: number) => {
+        setCurrentVideo(index);
+    };
+
+    const onPlayerStateChange = (index: number) => {
+        trackVideoClick(videos[index].analyticsLabel);
+    };
+
+    return (
+        <div className="video-carousel">
+            <div className="video-container">
+                {videos.map((video, index) => (
+                    <div key={video.id} style={{ display: index === currentVideo ? "block" : "none" }}>
+                        <lite-youtube
+                            videoid={video.id}
+                            style={{
+                                width: "320px",
+                                height: "180px",
+                            }}
+                            class="rounded-lg"
+                            playlabel={video.title}
+                            onClick={() => onPlayerStateChange(index)}
+                        ></lite-youtube>
+                    </div>
+                ))}
+            </div>
+            <div className="flex justify-center space-x-2 mt-2">
+                {videos.map((_, index) => (
+                    <button
```

Comment:
These buttons lack focus rings / indicators, making it hard to discern which one is to-be selected.

Reply:
Selected button shows as primary kumquat colour and on hover It changes to this (second dot)

![image](https://github.com/gitpod-io/gitpod/assets/55068936/6b552ec4-399f-4d94-a6e9-f051bc9dd21c)

Reply:
Sorry for the misunderstanding: the focus rings are missing for keyboard navigation, when tabbing through the buttons to select a video.

Reply:
Yeah, that would be good to have. But, I won’t consider this as a blocker for this PR. Maybe something for a follow-up?

Reply:
Resolved in [`caa2cd1`](https://github.com/gitpod-io/gitpod/pull/20021/commits/caa2cd1ff19da80c228fb7e143b43b72a3f808f0)

Reply:
It's indeed not a blocker, but I think this might actually be solved by simply removing the `focus:outline-none` on the trigger buttons on line 69. It might be a real letdown for folks who rely on the keyboard when using Gitpod, I say let's include them

Reply:
Yeah, I misunderstood your comment, it's already resolved in with our themed colour :)